easybuild/easyblocks/p/psmpi.py

```diff
@@ -29,12 +29,73 @@
 """
 
 import easybuild.tools.toolchain as toolchain
```

Comment:
@damianam While you're at it, please move down this import among the other `easybuild` import statements